### PR TITLE
Makes zombie smell through their eyes

### PR DESCRIPTION
**Pull request by MrJWhit**
## About The Pull Request

Gives zombie eyes the SEE_MOBS flag, the same as xenos, to let them "smell" living flesh.

## Why It's Good For The Game

More flavor to zombies, and a buff to our romerol friends.

## Changelog
:cl:
add: Added new things
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
Keep in mind this will also buff halloween roundstart zombies. If this isn't within your intention, I suggest you give the roundstart (the not spaceproof one) species a new subtype without the added SEE_MOBS flag.

**Comment by MrJWhit** [comment]
So, this actually bypasses the halloween zombies, because the only species datum that calls for their eyes (mutanteyes) is the infection zombie, which is the intended buff. 
The infection zombies includes the armor, speed, zombie hand, and everything that makes the romerol zombies fun.

**Comment by Ghommie** [comment]
Oh yeah, my bad, only romerol zombies have NV eyes, all fine then.

File: code/modules/surgery/organs/eyes.dm [hunk 1]
```diff
@@ -71,6 +71,7 @@
 /obj/item/organ/eyes/night_vision/zombie
 	name = "undead eyes"
 	desc = "Somewhat counterintuitively, these half-rotten eyes actually have superior vision to those of a living human."
+	sight_flags = SEE_MOBS
```
Comment (Trilbyspaceclone) [inline code/modules/surgery/organs/eyes.dm:74]: This affects all zombe types. I.e mobs as well

Comment (MrJWhit) [reply code/modules/surgery/organs/eyes.dm:74]: Only romerol zombies have those eyes.
datum/species/zombie/infectious are the only zombies that have the line `	mutanteyes = /obj/item/organ/eyes/night_vision/zombie`. The base zombie species, does not have that line.

Comment (Trilbyspaceclone) [reply code/modules/surgery/organs/eyes.dm:74]: Alright

**Review by silicons — APPROVED** [review]

**Pull request merged**